src/views/virtualmachines/actions/components/VirtualMachineMigration/tabs/VirtualMachineMigrationDetails.tsx

```diff
@@ -98,6 +99,7 @@ const VirtualMachineMigrationDetails: FC<VirtualMachineMigrationDetailsProps> =
         <Radio
           id="selected-volumes"
           isChecked={!allVolumes}
+          isDisabled={isEmpty(pvcs)}
```

Comment:
it seems we should also disable the other radio button - right now it looks as this option was valid  (but the wizard is blocked)